source/gsoc/gsoc-admin-tasks.md

```diff
@@ -23,7 +23,7 @@ contributors to your projects.
 If you want to participate with your project you now need to
 
  - understand what it means to be a mentor [1]
- - propose your project ideas. Just label your issues with gsoc2023 in Jira and
+ - propose your project ideas. Just label your issues with gsoc2024 in Jira and
    they will show up at [2]. See also [1].
```

Comment:
The `[2] https://s.apache.org/gsoc2011tasks` footnote is probably not useful anymore, either this page needs to be reworked to remove such precise references that need to be changed every year, or a warning added at the top of the page that details might not be valid anymore.